packages/next-frontend/src/components/connect-button/styles.module.css

```diff
@@ -0,0 +1,22 @@
+.connect_button__root {
```

Comment:
If we use CSS modules we can avoid using the old naming convention and go for a simpler styles. Class names will be mangled anyway and will be unique, so might as well go for the less verbose way..

Reply:
You're right, fixed in 553b93b44f72fa608f10241ebc55277f6270e045.